packages/Responsive/Responsive.md

```diff
@@ -1,5 +1,5 @@
 The `Responsive` component is a thin wrapper over the [**react-media**](https://github.com/ReactTraining/react-media) community
-component, which "listens for matches to a CSS media query and renders stuff based on whether the query matches or not."
+component, which "listens for matches to a CSS media query and renders stuff based on whether the query matches or not." It also implements [**sass-mq**](https://github.com/sass-mq/sass-mq) for an easy implementation of responsive behaviour through SASS.
```

Comment:
We may want to try splitting the docs into 2 sections so that we can clearly separate the responsive Sass stuff from the component docs.

In "styleguide.config.js" we could change the structure like this:

```js
// sections
{
  name: 'Responsiveness',
  content: path.resolve('packages/Responsive/docs/responsive.md'), // Sass docs
  components: path.resolve('packages/Responsive/Responsive.jsx') // component docs
}
```

Reply:
And if we do that, I'd structure the docs like this:

* First, a short, general info to what the responsive package provides.
* Then, show the breakpoint keys and the sizes/queries that they respond to.
* Then, document the Sass stuff
* Then, do the Responsive component and its docs.

Reply:
I agree with this, formatting it that way should reduce the need to reiterate the breakpoints. Plus  it'll be easier to push the new Sass stuff as our recommendation.